docs/integrations/destinations/databricks.md

```diff
@@ -2,46 +2,46 @@
 
 ## Overview
 
-This destination syncs data to Databricks cluster. Each stream is written to its own table.
+This destination syncs data to Databricks cluster. Each stream is written to its own [delta-table](https://delta.io/).
 
 This connector requires a JDBC driver to connect to Databricks cluster. By using the driver and the connector, you must agree to the [JDBC ODBC driver license](https://databricks.com/jdbc-odbc-driver-license). This means that you can only use this connector to connector third party applications to Apache Spark SQL within a Databricks offering using the ODBC and/or JDBC protocols.
 
 Currently, this connector requires 30+MB of memory for each stream. When syncing multiple streams, it may run into out-of-memory error if the allocated memory is too small. This performance bottleneck is tracked in [this issue](https://github.com/airbytehq/airbyte/issues/11424). Once this issue is resolved, the connector should be able to sync almost infinite number of streams with less than 500MB of memory.
 
 ## Sync Mode
 
-| Feature | Support | Notes |
-| :--- | :---: | :--- |
-| Full Refresh Sync | ✅ | Warning: this mode deletes all previously synced data in the configured bucket path. |
-| Incremental - Append Sync | ✅ |  |
-| Incremental - Deduped History | ❌ |  |
-| Namespaces | ✅ |  |
+| Feature                       | Support | Notes                                                                                |
+|:----------------------------- |:-------:|:------------------------------------------------------------------------------------ |
+| Full Refresh Sync             | ✅       | Warning: this mode deletes all previously synced data in the configured bucket path. |
+| Incremental - Append Sync     | ✅       |                                                                                      |
+| Incremental - Deduped History | ❌       |                                                                                      |
+| Namespaces                    | ✅       |                                                                                      |
```

Comment:
Nitpick. Personally I don't think formatting the markdown table like this is necessary. It can make the column too wide in the editor.

Reply:
Feel free to open a PR for it if you have the bandwidth :)

Reply:
@tuliren Thanks so much for the review. Yes, that's what I meant, I need to turn off my format, one of them did it automatically. I will remove them. 
It would be nice to align on a pretty printer for markdown, potentially also python (I guess we have for [Java](https://docs.airbyte.com/contributing-to-airbyte/code-style/)).

Reply:
If you are using IntelliJ, you can turn it off in `Preferences / Languages & Frameworks / Markdown / Automatic assistance in the editor`.